graylog2-server/src/test/java/org/graylog2/contentpacks/facades/InputFacadeTest.java

```diff
@@ -338,12 +369,89 @@ public void delete() throws NotFoundException {
                 .isInstanceOf(NotFoundException.class);
     }
 
+    @Test
+    @UsingDataSet(locations = "/org/graylog2/contentpacks/inputs.json", loadStrategy = LoadStrategyEnum.CLEAN_INSERT)
+    public void resolveNativeEntity() throws NotFoundException {
+        when(lookupuptableBuilder.lookupTable("whois")).thenReturn(lookupuptableBuilder);
+        when(lookupuptableBuilder.build()).thenReturn(lookupTable);
+        when(lookupTableService.newBuilder()).thenReturn(lookupuptableBuilder);
+        when(lookupTableService.hasTable("whois")).thenReturn(true);
+        when(dbLookupTableService.get("whois")).thenReturn(Optional.of(lookupTableDto));
+        when(lookupTableDto.id()).thenReturn("dead-beef");
+
+        final Input input = inputService.find("5ae2eb0a3d27464477f0fd8b");
+        EntityDescriptor entityDescriptor = EntityDescriptor.create(ModelId.of(input.getId()), ModelTypes.INPUT_V1);
+        EntityDescriptor expectedEntitiyDescriptor = EntityDescriptor.create(ModelId.of("dead-beef"), ModelTypes.LOOKUP_TABLE_V1);
+        Graph graph = facade.resolveNativeEntity(entityDescriptor);
+        assertThat(graph.nodes()).contains(expectedEntitiyDescriptor);
+    }
+
+    @Test
+    @UsingDataSet(locations = "/org/graylog2/contentpacks/inputs.json", loadStrategy = LoadStrategyEnum.CLEAN_INSERT)
+    public void resolveForInstallation() throws NotFoundException {
+        when(lookupuptableBuilder.lookupTable("whois")).thenReturn(lookupuptableBuilder);
+        when(lookupuptableBuilder.build()).thenReturn(lookupTable);
+        when(lookupTableService.newBuilder()).thenReturn(lookupuptableBuilder);
+        when(lookupTableService.hasTable("whois")).thenReturn(true);
+        when(dbLookupTableService.get("whois")).thenReturn(Optional.of(lookupTableDto));
+        when(lookupTableDto.id()).thenReturn("dead-beef");
+        final Input input = inputService.find("5ae2eb0a3d27464477f0fd8b");
+        final InputWithExtractors inputWithExtractors = InputWithExtractors.create(input, inputService.getExtractors(input));
+        final LookupTableExtractor extractor = (LookupTableExtractor) inputWithExtractors.extractors().iterator().next();
+        final ExtractorEntity extractorEntity = ExtractorEntity.create(
+                ValueReference.of(extractor.getTitle()),
+                ValueReference.of(extractor.getType()),
+                ValueReference.of(extractor.getCursorStrategy()),
+                ValueReference.of(extractor.getTargetField()),
+                ValueReference.of(extractor.getSourceField()),
+                ReferenceMapUtils.toReferenceMap(extractor.getExtractorConfig()),
+                Collections.emptyList(),
+                ValueReference.of(extractor.getConditionType()),
+                ValueReference.of(extractor.getConditionValue()),
+                ValueReference.of(extractor.getOrder())
+        );
+        List<ExtractorEntity> extractors = new ArrayList();
```

Comment:
Please make sure to use the generic array constructor variant here:

```suggestion
        List<ExtractorEntity> extractors = new ArrayList<>();
```

(Your IDE should warn you about that. :smiley:)